### DISCUSSION TITLE
Allow guest user Access in Sametime

### DISCUSSION THREAD

**Discussion by cormacmccarthyireland** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#16] [General]
Just a quick note that it is straight forward to allow Guest Access by default in Sametime Meetings by changing the (DOMI_scheduleSTMeeting) agent line (in either the mail template or an individual mailfile).

Just change the line as in the screenshot "AllowGuestAccess" to true (by default it's false).  It's probably a good note for documentation as people will ask.

![image](https://user-images.githubusercontent.com/82155444/114587585-b9fad280-9c7d-11eb-828e-5b8577de2662.png)

I've tested and it works perfectly for me.

**Comment by paulswithers** [comment]
Don't forget to make the corresponding change in DOMI_updateSTMeeting agent as well, otherwise updates will revert it to false.

**Comment by cormacmccarthyireland** [comment]
Thanks Paul. Done.
